plugins/nodes/src/js/pages/nodes-overview/HostsPageContent.js

```diff
@@ -164,36 +194,22 @@ class HostsPageContent extends React.Component {
       handleFilterChange,
       hosts,
       isFiltering,
-      nodeCount,
-      onResourceSelectionChange,
```

Comment:
For the props you removed here, why is it not removed from `HostsPageContent.propTypes` and the `<HostsPageContent />`, do we intend on using it later?

Reply:
forgot to remove them, will fix 👍